feconf.py

```diff
@@ -1393,7 +1393,7 @@ def get_empty_ratings():
     'required_attribute_names': [],
     'optional_attribute_names': [],
     'user_id_attribute_names': []
-}]
+}] # type: List[Dict[Text, Union[Text, List[Text], Dict[Text, List[Text]]]]]
```

Comment:
Could you use PEP 526 for variable types? See:
- https://www.python.org/dev/peps/pep-0526/#global-and-local-variable-annotations 
- https://stackoverflow.com/questions/39971929/what-are-variable-annotations

/cc @sajalasati @vojtechjelinek

Reply:
So sorry, I changed the storage files to py3 style, forgot this one. Sorry again!

Reply:
Done!